servant-client/test/Servant/ClientSpec.hs

```diff
@@ -30,7 +30,7 @@ module Servant.ClientSpec where
 import           Control.Applicative        ((<$>))
 #endif
 import           Control.Arrow              (left)
-import           Control.Concurrent         (forkIO, killThread, ThreadId)
+import           Control.Concurrent         (ThreadId, forkIO, killThread)
```

Comment:
I wouldn't make these changes at all. If you're not touching code, don't change formatting.